src/Components/Wishlist/Wishlist.css

```diff
@@ -0,0 +1,93 @@
+@import url("../../style.css");
+
+.product-container{
+    height: 34rem;
+    width: 79.8rem;
```

Comment:
try to give values in multiple of 4 or 8 ita a good practise to follow like 80 rem or so

Reply:
> try to give values in multiple of 4 or 8 ita a good practise to follow like 80 rem or so
 hey @i-am-a-codegirl   can you tell me little bit in brief about this  i am not getting your point why we need to multiple of 4 or 8